msg/VehicleLocalPosition.msg

```diff
@@ -38,11 +38,14 @@ float32 ay        # East velocity derivative in NED earth-fixed frame, (metres/s
 float32 az        # Down velocity derivative in NED earth-fixed frame, (metres/sec^2)
 
 float32 heading				# Euler yaw angle transforming the tangent plane relative to NED earth-fixed frame, -PI..+PI,  (radians)
+float32 heading_var
 float32 unaided_heading                 # Same as heading but generated by integrating corrected gyro data only
 float32 delta_heading			# Heading delta caused by latest heading reset [rad]
 uint8 heading_reset_counter		# Index of latest heading reset
 bool heading_good_for_control
 
+float32 tilt_var
```

Comment:
I'll merge for now, but I think we should move this to vehicle_attitude instead.

Reply:
I convinced myself that it had its place in there because it's the uncertainty of the attitude defined in the local frame, but you're probably right